lesson02/assignment/css/_innerbox.scss

```diff
@@ -0,0 +1,20 @@
+.inner-box {
+  padding: 2rem;
+  text-align: center;
+  display:flex;
+  flex-direction: row;
+  background-color: $white;
+  box-sizing:border-box;
+  white-space: nowrap;
+
+  .inner-most-box {
+    //tried to center the linkedIn logo but had a difficult time doing so, also left no resizing, didn't think it really had much impact.
```

Comment:
One thing you can do is look at using flexbox's `align-items: center`. Check out this cheatsheet for other helpful flexbox tools: [https://css-tricks.com/snippets/css/a-guide-to-flexbox/](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)